Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/core/VaccinationMigration.kt

```diff
@@ -0,0 +1,34 @@
+package de.rki.coronawarnapp.covidcertificate.vaccination.core
+
+import de.rki.coronawarnapp.covidcertificate.vaccination.core.repository.storage.StoredVaccinationCertificateData
+import de.rki.coronawarnapp.covidcertificate.vaccination.core.repository.storage.VaccinationStorage
+import timber.log.Timber
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class VaccinationMigration @Inject constructor(private val vaccinationStorage: VaccinationStorage) {
+
+    suspend fun doMigration(): Set<StoredVaccinationCertificateData> {
```

Comment:
yes

Reply:
But we do have to migrate `lastSeenBoosterRuleIdentifier` ? Otherwise we loose the information which booster notification the user has already seen and the user would get another notification after update, or am I wrong?

Reply:
yes, that's true, but it was communicated in the beginning, and it's an acceptable issue

Reply:
I can remember that it is acceptable when persons are split up because of the new grouping. Is it acceptable here too?

Reply:
It wouldn't be too much effort to migrate `lastSeenBoosterRuleIdentifier` from `VaccinatedPersonData` to `PersonCertificateSettings`, or?